data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/model/dto/orgdetails/Address.java

```diff
@@ -0,0 +1,64 @@
+package gov.cdc.etldatapipeline.organization.model.dto.orgdetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgElastic;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgReporting;
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ToString(callSuper = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class Address implements OrgExtension {
+    private String addrElpCd;
+    private String addrElpUseCd;
+    private Long addrPlUid;
+    private String streetAddr1;
+    private String streetAddr2;
+    private String city;
+    private String zip;
+    private String cntyCd;
+    private String state;
+    private String cntryCd;
+    private String state_desc;
+    private String county;
+    private String withinCityLimitsInd;
+    private String country;
+    private String addressComments;
+
+    public <T> T updateOrg(T org) {
```

Comment:
Does the addr_elp_cd, addr_elp_use_cd and addr_pl_uid get pulled in via this section?
There is a check for addr_elp_use_cd = 'WP', addr_elp_cd = 'O'. ELP class_cd ='PST' is handled in the sproc.

Reply:
Updated sproc.